index.html

```diff
@@ -130,25 +130,66 @@ <h3>Open Source Programs we are a part of </h3>
+                            <p>Phone Number (optional)</p>
+                            <input type="number" class="contact-input">
+                            
+                        <!-- START CAPTCHA -->
                             <div class="capbox">
                                 <div id="CaptchaDiv"></div>
                                     <div class="capbox-inner">
-                                        Type the number <br>
+                                        Captcha (Type the number)  <br>
                                         <input type="hidden" id="txtCaptcha">
                                         <input type="text" name="CaptchaInput" id="CaptchaInput" size="15"><br>
                                     </div>
                             </div>
                             <br><br>
                             <!-- END CAPTCHA -->
+
+                            <label>
+                                
+                                <p>Rate Us</p>
+                                <br>
+                                <div class="container-emoji">
+                                  <div class="item">
+                                    <label for="0">
+                                      <input class="radio" type="radio" name="feedback" id="0" value="0">
+                                      <span>🤬</span>
+                                    </label>
+                                  </div>
+                                  <div class="item">
+                                    <label for="1">
+                                      <input class="radio" type="radio" name="feedback" id="1" value="1">
+                                      <span>🙁</span>
+                                    </label>
+                                  </div>
+                                  <div class="item">
+                                    <label for="2">
+                                      <input class="radio" type="radio" name="feedback" id="2" value="2">
+                                      <span>😶</span>
+                                    </label>
+                                  </div>
+                                  <div class="item">
+                                    <label for="3">
+                                      <input class="radio" type="radio" name="feedback" id="3" value="3">
+                                      <span>😁</span>
+                                    </label>
+                                  </div>
+                                  <div class="item">
+                                    <label for="4">
+                                      <input class="radio" type="radio" name="feedback" id="4" value="4">
+                                      <span>😍</span>
+                                    </label>
+                                  <div>
+                              </div>
+                            </label><br><br>
```

Comment:
Please ensure that you indent code blocks properly. In the long term, it messes up the readability of code